ArduPlane/quadplane.cpp

```diff
@@ -354,9 +354,9 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     AP_SUBGROUPPTR(loiter_nav, "LOIT_",  2, QuadPlane, AC_Loiter),
 
     // @Param: TAILSIT_THSCMX
-    // @DisplayName: Maximum control throttle scaling value
-    // @Description: Maximum value of throttle scaling for tailsitter velocity scaling, reduce this value to remove low thorottle D ossilaitons 
-    // @Range: 1 5
+    // @DisplayName: Maximum gain boost or attenuation based on throttle and attitude
+    // @Description: If > 1, maximum gain attenuation at high throttle/tilt angle, if > 1, max gain boost at low throttle
+    // @Range: 0.1 5
     // @User: Standard
     AP_GROUPINFO("TAILSIT_THSCMX", 3, QuadPlane, tailsitter.throttle_scale_max, 5),
```

Comment:
might as well take the opportunity to change the default, I think 1.5 or 2 would be better.

Reply:
good point, 2 sounds good to me